packages/components/src/focal-point-picker/styles/focal-point-picker-style.ts

```diff
@@ -91,29 +104,29 @@ export const GridView = styled.div`
 	position: absolute;
 	top: 50%;
 	transform: translate3d( -50%, -50%, 0 );
-	transition: opacity 120ms linear;
+	transition: opacity 100ms linear;
 	z-index: 1;
 
 	opacity: ${ ( { showOverlay }: { showOverlay?: boolean } ) =>
 		showOverlay ? 1 : 0 };
 `;
 
 export const GridLine = styled.div`
-	background: white;
-	box-shadow: 0 0 2px rgba( 0, 0, 0, 0.6 );
+	background: rgba( 255, 255, 255, 0.4 );
+	backdrop-filter: blur( 16px ) saturate( 200% );
 	position: absolute;
-	opacity: 0.4;
+	opacity: 1;
```

Comment:
Isn't it okay to delete this property itself?

Reply:
Yup, applied in https://github.com/WordPress/gutenberg/pull/58459/commits/13bc32959aa67b6ae559168c35dd698abd3cccae.